src/ui/BackButton/BackButton.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { withDesign } from 'storybook-addon-designs';
```

Comment:
Let's leave out `withDesign` unless we're actually using it.

Reply:
Sounds good, removed with https://github.com/coordinape/coordinape/pull/1595/commits/3b733921529d2f7f73f9b810ede7160aaeab6525